# Copyright (C) 2009  Miriam Ruiz <little_miry@yahoo.es>
# Distributed under the same license as the game (GPL3+). See debian/copyright

Index: pink-pony-1.4.1/pony.options
===================================================================
--- pink-pony-1.4.1.orig/pony.options
+++ pink-pony-1.4.1/pony.options
@@ -9,7 +9,7 @@ polygon_mode = fill;
 
 // Resource directory
 
-resource_dir = "resources/";
+resource_dir = "/usr/share/pink-pony/";
 
 // Particles
 
Index: pink-pony-1.4.1/src/main.cc
===================================================================
--- pink-pony-1.4.1.orig/src/main.cc
+++ pink-pony-1.4.1/src/main.cc
@@ -20,7 +20,7 @@ Config config;
 
 int main(int argc, char** argv)
 {
-    string config_file = "pony.options";
+    string config_file = "/usr/share/pink-pony/pony.options";
     bool running = true;
     
     if (argc > 1) {
